pyensembl/ensembl_release.py

```diff
@@ -120,6 +123,33 @@ def run_query():
             return results
         return cached_object(pickle_path, compute_fn=run_query)
 
+    def download_all(self):
+        self.download_annotations()
+        self.download_transcripts()
+
+    def download_annotations(self):
+        if self.db.connection_if_exists():
+            print ("Annotation data for release %s is already "
+                   "downloaded and installed" % self.release)
+            return
+        self.db.create_database()
+        print ("Annotation data for release %s has been downloaded "
+               "and installed" % self.release)
+
+    def download_transcripts(self):
+        cache = self.reference.cache
+        if cache.exists(self.reference.url,
```

Comment:
Shouldn't this logic be on the ReferenceTranscripts object? All this `self.reference.foo` seems like a code smell (or a hint that this method should move elsewhere).